code-of-conduct.md

```diff
@@ -0,0 +1,62 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of
+fostering an open and welcoming community, we pledge to respect all people who
+contribute through reporting issues, posting feature requests, updating
+documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
+experience for everyone, regardless of level of experience, gender, gender
+identity and expression, sexual orientation, disability, personal appearance,
+body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic
+  addresses, without explicit permission
+* Other unethical or unprofessional conduct
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+By adopting this Code of Conduct, project maintainers commit themselves to
+fairly and consistently applying these principles to every aspect of managing
+this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+A working group of community members is committed to promptly addressing any
+reported issues. The working group is made up of MDS V students enrolled in DSCI_522 
+at the University of British Columbia.Instances of abusive, harassing, or otherwise 
+unacceptable behavior may be reported by contacting the working group by e-mail 
+(nphaterp@student.ubc.ca). Messages sent to this e-mail address will not be 
+publicly visible but only to the working group members. 
+The working group currently includes
+
+- Fatime Selimi
+- Nicholas Wu
+- Tanmay Sharma
+- Neel Phaterpekar
+
+All complaints will be reviewed and investigated and will result in a response
+that is deemed necessary and appropriate to the circumstances. Maintainers are
+obligated to maintain confidentiality with regard to the reporter of an
+incident.
+
+This Code of Conduct is copied from the [pandas-governance Code of Conduct](https://github.com/pandas-dev/pandas-governance/blob/master/code-of-conduct.md), which was originally adapted from the [Contributor Covenant][homepage],
```

Comment:
This is minor but should we say "This Code of Conduct is **adopted** from..."? We have our names in this document which makes it slightly different from the one you're linking here.

Reply:
This is the link I went to https://github.com/pandas-dev/pandas-governance/blob/master/code-of-conduct.md. I think adapted is still the correct word (they too used it)